liblwgeom/liblwgeom_internal.h

```diff
@@ -488,5 +489,6 @@ int ptarray_npoints_in_rect(const POINTARRAY *pa, const GBOX *gbox);
 int gbox_contains_point2d(const GBOX *g, const POINT2D *p);
 int lwpoly_contains_point(const LWPOLY *poly, const POINT2D *pt);
 POINT4D* lwmpoint_extract_points_4d(const LWMPOINT* g, uint32_t* npoints, int* input_empty);
+char* lwstrdup(const char* a);
```

Comment:
Nice addition, but since you added it, can you also change the existing call to strdup (which is a bug)?
`liblwgeom/g_box.c:397:          return strdup("NULL POINTER");`

Reply:
Nevermind, I see that you've already done it.